packages/server/lib/server.ts

```diff
@@ -68,7 +68,14 @@ const webAuth =
         ? [passport.authenticate('basic', { session: false }), authMiddleware.basicAuth.bind(authMiddleware)]
         : [authMiddleware.noAuth.bind(authMiddleware)];
 
-app.use(express.json({ limit: '75mb' }));
+app.use(
+    express.json({
+        limit: '75mb',
```

Comment:
this feels like a very high limit that has to fit in memory and for each request, is there any reason for that?

Reply:
75mb seems a lot, even 1mb for a sync file seems like a lot but we'll see 👌🏻

Reply:
It’s multiple files. It was at 50mb and the limit was hit then I believe.

Reply:
😱  alright